src/devices/Vl53L1X/Vl53L1X.cs

```diff
@@ -0,0 +1,976 @@
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the intermeasurement period in ms.
+        /// Intermeasurement period must be >/= timing budget. This condition is not checked by the API,
+        /// the customer has the duty to check the condition. Default = 100 ms.
+        /// </summary>
+        public ushort InterMeasurementInMs
+        {
+            get
+            {
+                uint temp = ReadUInt32((ushort)Registers.VL53L1X_SYSTEM__INTERMEASUREMENT_PERIOD);
+                uint clockPll = ReadUInt32((ushort)Registers.VL53L1X_RESULT__OSC_CALIBRATE_VAL);
+
+                clockPll &= 0x3FF;
+
+                return (ushort)(temp / (clockPll * 1.065));
+            }
+
+            set
+            {
+                uint clockPll = ReadUInt32((ushort)Registers.VL53L1X_RESULT__OSC_CALIBRATE_VAL);
+                clockPll &= 0x3FF;
+                WriteUInt32((ushort)Registers.VL53L1X_SYSTEM__INTERMEASUREMENT_PERIOD,
+                    (uint)(clockPll * value * 1.075));
+            }
+        }
+
+        /// <summary>
+        /// Gets the <see cref="BootState" /> of the device (1 = booted, 0 = not booted).
+        /// </summary>
+        public BootState BootState
+        {
+            get
+            {
+                byte bootStateVal = ReadByte((ushort)Registers.VL53L1X_FIRMWARE__SYSTEM_STATUS);
+                return
+                    bootStateVal > 0
+                        ? BootState.Booted
+                        : BootState
+                            .NotBooted;
+            }
+        }
+
+        /// <summary>
+        /// Gets the sensor ID which must be 0xEEAC.
+        /// </summary>
+        public ushort SensorId
+        {
+            get
+            {
+                return ReadUInt16((ushort)Registers.VL53L1X_IDENTIFICATION__MODEL_ID);
+            }
+        }
+
+        /// <summary>
+        /// Returns the signal per SPAD in kcps/SPAD.
+        /// </summary>
+        public ushort SignalPerSpad
+        {
+            get
+            {
+                ushort signal =
+                    ReadUInt16((ushort)Registers.VL53L1X_RESULT__PEAK_SIGNAL_COUNT_RATE_CROSSTALK_CORRECTED_MCPS_SD0);
+                ushort spNb = ReadUInt16((ushort)Registers.VL53L1X_RESULT__DSS_ACTUAL_EFFECTIVE_SPADS_SD0);
+                return (ushort)(2000.0 * signal / spNb);
+            }
+        }
+
+        /// <summary>
+        /// Returns the ambient per SPAD in kcps/SPAD.
+        /// </summary>
+        public ushort AmbientPerSpad
+        {
+            get
+            {
+                ushort ambientRate = ReadUInt16((ushort)Registers.RESULT__AMBIENT_COUNT_RATE_MCPS_SD);
+                ushort spNb = ReadUInt16((ushort)Registers.VL53L1X_RESULT__DSS_ACTUAL_EFFECTIVE_SPADS_SD0);
+                return (ushort)(2000.0 * ambientRate / spNb);
+            }
+        }
+
+        /// <summary>
+        /// Returns the signal in kcps.
```

Comment:
what is kcps stand for?

Reply:
It stands for kilo counts per second. I've added remarks to the comments to explain the unit.